acrobat/scripts/utils.js

```diff
@@ -35,3 +35,56 @@ export const [setLibs, getLibs] = (() => {
     }, () => libs,
   ];
 })();
+
+export const getBrowserData = () => {
```

Comment:
wondering if we need to break down browser by name or if we can rely on simpler logic (see other comments)